controllers/hospital.controller.js

```diff
@@ -0,0 +1,120 @@
+const Hospital = require('../models/hospital.model');
+
+exports.addHospital = async (req, res) => {
+
+    const new_hospital = {
+        name: req.body.name,
+        address: req.body.address,
+        doctor_ids: req.body.doctor_ids
+    };
+
+    try{
+        const added_hospital = await Hospital.create( new_hospital ); 
+        return res.status(200).send({
```

Comment:
201 is the Http status for creation.

201 itself means, that resource is created